Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/clip.h

```diff
@@ -688,6 +753,7 @@ void split(TriangleMesh& tm,
   *
   * \note In the current implementation it is not possible to set the vertex point map and the default will be used. `Iso_cuboid_3` must be
   * from the same %Kernel as the point of the vertex point map.
```

Comment:
@sloriot can you please comment on what that note means? The mesh's vpm seems to be used just fine. Does this refer to an hypothetical VPM associated to the iso cuboid?